packages/gatsby-theme/src/components/Share/index.js

```diff
@@ -0,0 +1,18 @@
+import Share from "./Share"
+import ShareHeading from "./ShareHeading"
+import ShareFacebook from "./ShareFacebook"
+import ShareLinkedin from "./ShareLinkedin"
+import ShareReddit from "./ShareReddit"
+import ShareTwitter from "./ShareTwitter"
+import ShareWhatsapp from "./ShareWhatsapp"
+import ShareContainer from "./ShareContainer"
+
+Share.Heading = ShareHeading
+Share.Facebook = ShareFacebook
+Share.Linkedin = ShareLinkedin
+Share.Reddit = ShareReddit
+Share.Twitter = ShareTwitter
+Share.Whatsapp = ShareWhatsapp
+Share.Container = ShareContainer
+
+export default Share
```

Comment:
We should add this component to the Index of the components
`packages/gatsby-theme/src/components/index.js`